schema/filter_artworks.js

```diff
@@ -23,10 +23,16 @@ export const FilterArtworksType = new GraphQLObjectType({
       type: GraphQLInt,
       resolve: ({ aggregations }) => aggregations.total.value,
     },
+    followed_artists_total: {
+      type: GraphQLInt,
+      resolve: ({ aggregations }) => aggregations.followed_artists.value,
+    },
```

Comment:
Well, the current total we just expose in the schema, so this just follows that pattern.